dp_mobility_report/report/report.py

```diff
@@ -68,99 +68,194 @@ def report_elements(mdreport: "MobilityDataReport") -> dict:
         total=4, desc="Create report", disable=mdreport.disable_progress_bar
     ) as pbar:
```

Comment:
we should have a look into the budget splitting: as not all analyses are conducted when there is no time component, the budget needs to be split correctly between the remaining analyses